pipeline.py

```diff
@@ -180,8 +185,21 @@
     # traced data log.
     if drive_upload:
         print("Uploading CSVs to Google Drive...")
-        drive_client_wrapper.init_client(drive_credentials_path)
 
+        # Fetch the Rapid Pro Token from the Google Cloud Storage URL
+        parsed_rapid_pro_token_file_url = urlparse(pipeline_configuration.drive_credentials_file_url)
+        bucket_name = parsed_rapid_pro_token_file_url.netloc
+        blob_name = parsed_rapid_pro_token_file_url.path.lstrip("/")
+
+        print(f"Downloading Drive service account credentials from file '{blob_name}' in bucket '{bucket_name}'...")
```

Comment:
consider s/file/blob

files are usually local

Reply:
(nice logging though) 👍

Reply:
changed, both here and in 2_fetch_raw_data.py